glDisplay/docs/glDisplay.rst

```diff
@@ -33,7 +33,7 @@ An instance of this plugin has the following parameters:
 **color**: {int}
     0: Red, 1: Green, 2: Blue, 3: White
 **currentIdx**: {int}
-    Index of currently displayed image [0..)
+    Index of currently displayed image [0..]
```

Comment:
The previous version was correct, since ( and [ are interpreted in a mathematical sense: [0..10) means that 0 is included, but 10 is excluded

Reply:
Thank you. Fixed it.